code/modules/admin/admin_verbs.dm

```diff
@@ -585,15 +585,15 @@ GLOBAL_LIST_INIT(roundstart_mod_verbs, list(
 	message_admins("[key_name(usr)] announced a random fact.")
 	SSticker.mode?.declare_fun_facts()
 
-/client/proc/toggle_ares_ping()
-	set name = "Toggle ARES notification sound"
-	set category = "Preferences.Logs"
+/client/proc/toggle_admin_pings()
+	set name = "Toggle StaffIC log sounds"
```

Comment:
aaa

Reply:
wut

Reply:
I don't understand :sob:

Reply:
Sorry i just meant it's lacking a space, whatever